setup.json

```diff
@@ -7,25 +7,28 @@
     "license": "MIT",
     "classifiers": [
         "Environment :: Plugins",
-        "Programming Language :: Python :: 3.6",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
         "Intended Audience :: Science/Research",
         "License :: OSI Approved :: MIT License",
         "Framework :: AiiDA"
     ],
-    "version": "0.2.0",
+    "version": "1.0.0b",
     "setup_requires": ["reentry"],
     "install_requires": [
-        "aiida_core[atomic_tools] >= 1.0.0, <2.0.0",
-        "cclib"
+        "aiida_core[atomic_tools] >= 1.6.8, <2.0.0",
+        "periodictable==1.6.1"
     ],
     "entry_points": {
         "aiida.calculations": [
-            "orca = aiida_orca.calculations:OrcaCalculation"
+            "orca_main = aiida_orca.calculations:OrcaCalculation",
```

Comment:
This would be a breaking change, but we should consider renaming this to `orca.main` to make it in line with other plugins. I've been bitten by this multiple times. :-)

Reply:
I agree.

Reply:
It is resolved. I decided to go with `orca.{name of executable}` after all.

Reply:
Thanks, that's a good change.